easybuild/easyconfigs/v/Valgrind/Valgrind-3.15.0-foss-2020a.eb

```
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Valgrind'
+version = '3.15.0'
+
+homepage = 'https://valgrind.org'
+description = "Valgrind: Debugging and profiling tools"
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Surely, valgrind doesn't care about LAPACK, so, we should be able to drop this down to GCC level?
```suggestion
toolchain = {'name': 'GCC', 'version': '9.3.0'}
```

---

makes sense

---

pushed a new commit